### DISCUSSION TITLE
Client library for GoTrue in Go

### DISCUSSION THREAD

**Discussion by kwoodhouse93** [supabase/supabase#10142] [Show and tell]
Hi! I've written a client library in Golang for GoTrue!

Check it out here:
https://github.com/kwoodhouse93/gotrue-go

I was building a backend server in Go with Supabase and I found myself wanting to make use of the awesome features in Supabase Auth (because auth is hard!). I didn't need a full Supabase client, but I still found myself writing a little library for interacting with GoTrue, just to avoid sprinkling a load of HTTP requests into my backend logic.

I figured this could be useful to others, so I went back and added the endpoints I haven't used, and packaged it up into this repo. I hope somebody else finds it useful!

It's fairly well tested in CI, but I'll be testing more of it out in my own project over the coming weeks.

It's released under the MIT license, and I'm open to any feedback, issues or contributions!

**Comment by thorwebdev** [comment]
Thanks for sharing this @kwoodhouse93, it looks great. As discussed, we're happy to move it to the `supabase-community` org to help others discover and use it :)
